src/analysis/normed_space/spectrum.lean

```diff
@@ -87,11 +88,29 @@ metric.is_compact_of_is_closed_bounded (is_closed a) (is_bounded a)
 
 theorem spectral_radius_le_nnnorm (a : A) :
   spectral_radius 𝕜 a ≤ ∥a∥₊ :=
+by { refine bsupr_le (λ k hk, _), exact_mod_cast norm_le_norm_of_mem hk }
+
+open ennreal polynomial
+
+theorem spectral_radius_le_pow_nnnorm_pow_one_div (a : A) (n : ℕ) :
+  spectral_radius 𝕜 a ≤ ∥a ^ (n + 1)∥₊ ^ (1 / (n + 1) : ℝ) :=
 begin
-  suffices h : ∀ k ∈ σ a, (∥k∥₊ : ℝ≥0∞) ≤ ∥a∥₊,
-  { exact bsupr_le h, },
-  { intros _ hk,
-    exact_mod_cast norm_le_norm_of_mem hk },
+  refine bsupr_le (λ k hk, _),
+  /- apply easy direction of the spectral mapping theorem for polynomials -/
+  have pow_mem : k ^ (n + 1) ∈ σ (a ^ (n + 1)),
+    by simpa only [one_mul, algebra.algebra_map_eq_smul_one, one_smul, aeval_monomial, one_mul,
+      eval_monomial] using subset_polynomial_aeval a (monomial (n + 1) (1 : 𝕜)) ⟨k, hk, rfl⟩,
```

Comment:
```suggestion
    by simpa using subset_polynomial_aeval a (monomial (n + 1) (1 : 𝕜)) ⟨k, hk, rfl⟩,
```
I find proofs more readable with more automation, so I tend not to squeeze my terminal simps.  This is a personal stylistic choice though (some people prefer the marginal greater efficiency of the squeezed version), so do as you prefer.

Reply:
Besides just the greater efficiency (which I've decided matters to me because my machine is a decade old :-) ), there is also the opportunity to visually interrogate `simp` when you have used `simp only`. Of course, you can always do `simp?`, but then you have to wait ... again.